config/default.schema.json

```diff
@@ -247,6 +254,29 @@
 			"description": "Synchronous encryption key used to secure storage provider secrets. Only used with FEATURE_MULTIPLE_S3_PROVIDERS_ENABLED=true",
 			"pattern": ".{16}.*"
 		},
+		"STORAGE_SIGNED_URL_EXPIRE": {
+			"type": "integer",
+			"default": 60,
+			"description": "Expiration time in seconds for signedUrl used by antivirus"
+		},
+		"ENABLE_FILE_SECURITY_CHECK": {
+			"type": "boolean",
+			"description": "enables antivirus check for files",
+			"default": false
+		},
+		"FILE_SECURITY_CHECK_SERVICE_URI": {
+			"type": "string",
+			"description": "Antivirus file endpoint",
+			"default": "http://localhost:8081/scan/file"
```

Comment:
move this to development.json the default should not be defined for production/test